Source/init.cpp

```diff
@@ -156,12 +156,12 @@ void __fastcall init_create_window(int nCmdShow)
 	wcex.style = CS_HREDRAW | CS_VREDRAW;
 	wcex.lpfnWndProc = WindowProc;
 	wcex.hInstance = ghInst;
-	wcex.hIcon = LoadIcon(ghInst, MAKEINTRESOURCE(IDI_ICON1));
-	wcex.hCursor = LoadCursor(0, IDC_ARROW);
+	wcex.hIcon = LoadIcon(ghInst, (LPCSTR)MAKEINTRESOURCE(IDI_ICON1));
+	wcex.hCursor = LoadCursor(0, (LPCSTR)IDC_ARROW);

```

Comment:
Try and correct the value of https://github.com/diasurgical/devilutionX/blob/22a3657ee47c5f757a028b4bcd82cd056866c05c/SourceS/miniwin.h#L280 instead